src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $size-gap;
+  padding: 10px;
+  max-width: $size-box-day * 7 + $size-gap * 6;
+
+  @each $day, $margin in $days {
+    &--start-day_#{$day} :first-child {
+      margin-left: $margin * ($size-box-day + $size-gap);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length_#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    @include square($size-box-day);
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+    background-color: #eee;
```

Comment:
not fixed 
![image](https://github.com/mate-academy/layout_calendar/assets/70009205/5dbc7f42-a808-4913-81a4-e8cfef82ae10)